exsclaim/journal.py

```diff
@@ -615,4 +624,81 @@ def find_captions(self, figure):
     def save_figure(self, figure_name, image_url):
         figures_directory = self.results_directory / "figures"
         out_file = figures_directory / figure_name
-        urllib.request.urlretrieve(image_url, out_file)
\ No newline at end of file
+        urllib.request.urlretrieve(image_url, out_file)
+
+
+class Wiley(JournalFamily):
+    domain =            "https://onlinelibrary.wiley.com"
+    search_path =       "/action/doSearch?"
+    page_param =        "startPage="
+    max_page_size =     "pageSize=20"
+    term_param =        "AllField="
+    order_param =       "sortBy="
+    open_param =        "ConceptID=15941"
```

Comment:
such a strange parameter

Reply:
I agree, I tried to figure out if the ConceptID=15941 was an artifact or page specific but I reproduced it every time I searched for open journals.